lib/src/ui/codelessly_widget.dart

```diff
@@ -308,6 +316,35 @@ class _CodelesslyWidgetState extends State<CodelesslyWidget> {
         );
   }
 
+  @override
+  void didChangeDependencies() {
+    super.didChangeDependencies();
+
+    // Only run this if controller is initialized and everything is ready.
+    // This shouldn't run for the first time, only when media query changes.
```

Comment:
Please clarify why this shouldn't run for the first time. What problem is this solving? Describe it here.